lib/conemesh.js

```diff
@@ -718,12 +728,23 @@ proto.pick = function(pickData) {
   var cell      = this.cells[cellId]
   var pos =     this.positions[cell[1]].slice(0, 3)
 
-  return {
-    // corresponding to input indices
-    index: Math.floor(cell[1] / 48),
+  var result = {
     position: pos,
-    dataCoordinate: pos
+    dataCoordinate: pos,
+    index: Math.floor(cell[1] / 48)
+  }
+
+
+  if(this.traceType === 'cone') {
+    result.index = Math.floor(cell[1] / 48)
+  } else if(this.traceType === 'tube') {
```

Comment:
Can we call this `streamtube` instead for consistency. This should make debugging in the future much easier.

Reply:
Good call. Thanks for your review!

Reply:
Done in https://github.com/gl-vis/gl-cone3d/pull/17/commits/bb698b70b617ca4e90414587448f6d875f9c7076